libs/pages/services/src/lib/ui/container/container.tsx

```diff
@@ -28,34 +28,73 @@ export interface ContainerProps {
   statusActions: StatusMenuActions[]
   environment?: EnvironmentEntity
   children?: React.ReactNode
+  removeEnvironment?: () => void
 }
 
 export function Container(props: ContainerProps) {
-  const { environment, children, statusActions } = props
+  const { environment, children, statusActions, removeEnvironment } = props
   const { organizationId, projectId, environmentId } = useParams()
   const location = useLocation()
 
   const copyContent = `Organization ID: ${organizationId}\nProject ID: ${projectId}\nEnvironment ID: ${environmentId}`
 
-  const buttonActionsDefault = [
-    {
-      iconLeft: <Icon name="icon-solid-play" className="px-0.5" />,
-      iconRight: <Icon name="icon-solid-angle-down" className="px-0.5" />,
-      menusClassName: 'border-r border-r-element-light-lighter-500',
-      statusActions: {
-        status: environment?.status && environment?.status.state,
-        actions: statusActions,
-      },
-    },
-    {
-      iconLeft: <Icon name="icon-solid-scroll" className="px-0.5" />,
-      onClick: () =>
-        window.open(
-          `https://console.qovery.com/platform/organization/${organizationId}/projects/${projectId}/environments/${environmentId}/applications?fullscreenLogs=true`,
-          '_blank'
-        ),
-    },
-  ]
+  const buttonActionsDefault = () => {
+    if (removeEnvironment) {
```

Comment:
You duplicate the code for to update the status, you can directly add a conditon for the removeEnvironment object